schemas/pug-lintrc-schema.json

```diff
@@ -0,0 +1,456 @@
+{
```

Comment:
We use `pug-lint` from workspace or global `node_modules` directory. VS Code can't use these files, because it's not a static files (we don't know path in advance).